nuget/OpenSans/Uno.Fonts.OpenSans/buildTransitive/net7.0/Uno.Fonts.OpenSans.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
+    <Target Name="UnoRemoveUnusedOpenSans" AfterTargets="_UnoAddLibraryAssets">
```

Comment:
This target is not yet tested. I'll test everything together once we have the single font file.